core/new-gui/src/app/workspace/component/result-panel/result-panel.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="texera-workspace-result-panel-body texera-workflow-component-body">
-  <div *ngIf="showMessage">
+  <div class="error-message" *ngIf="showMessage">
```

Comment:
change the class name to be a more specific name, prefix it with `texera-`, such as `texera-result-panel-message`
This is to minimize the conflict of name of our component with other library's name. Another library could also have a class name called "error-message".